paig-server/backend/paig/api/shield/scanners/GuardrailScanner.py

```diff
@@ -0,0 +1,102 @@
+import logging
+import boto3
+import os
+
+from api.shield.scanners.BaseScanner import Scanner
+
+logger = logging.getLogger(__name__)
+
+
+class GuardrailScanner(Scanner):
+    """
+    Scanner implementation for applying guard rail policies in the input prompt.
+    """
+
+    def __init__(self, name, request_types, enforce_access_control, model_path, model_score_threshold, entity_type,
```

Comment:
can you add data type for all variables and python docs, it will be useful in type hints

Reply:
updated the method signate to use kwargs and added python docs